<ui version="4.0">
 <class>PetInfoWindow</class>
 <widget class="QMainWindow" name="PetInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pet Info Login</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <!-- Radio buttons for animal types -->
    <item>
     <layout class="QHBoxLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="animalLabel">
        <property name="text">
         <string>What animals do you sell?</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioDogs">
        <property name="text">
         <string>Dogs</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioCats">
        <property name="text">
         <string>Cats</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioBoth">
        <property name="text">
         <string>Both</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <!-- Radio buttons for vaccination status -->
    <item>
     <layout class="QHBoxLayout">
     <item row="5" column="0">
       <widget class="QLabel" name="labelVaccination">
        <property name="text">
         <string>Are they all vaccinated and allergy free?</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QRadioButton" name="radioYes">
        <property name="text">
         <string>Yes</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QRadioButton" name="radioNo">
        <property name="text">
         <string>No</string>
        </property>
       </widget>
       </item>
       <item row="8" column="0">
        <widget class="QRadioButton" name="radioSome">
         <property name="text">
          <string>Some</string>
         </property>
        </widget>
       </item>
     </layout>
    </item>
    <!-- Checklist for age ranges -->
    <item>
     <layout class="QHBoxLayout">
     <item row="10" column="0">
      <widget class="QLabel" name="labelAge">
       <property name="text">
        <string>Select all that apply</string>
       </property>
      </widget>
      </item>
      <item row="11" column="0">
       <widget class="QCheckBox" name="check0_2Years">
        <property name="text">
         <string>0-2 years</string>
        </property>
       </widget>
       </item>
       <item row="12" column="0">
        <widget class="QCheckBox" name="check3_5Years">
         <property name="text">
          <string>3-5 years</string>
         </property>
        </widget>
        </item>
        <item row="13" column="0">
         <widget class="QCheckBox" name="check5Years">
          <property name="text">
           <string>More than 5 years</string>
          </property>
        </widget>
        </item>
     </layout>
    </item>
    <!-- Spacer -->
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <!-- Submit and Cancel buttons -->
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="text">
       <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSubmit">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <resources/>
  <connections/>
 </ui>
